cmd/poller/plugin/plugin.go

```diff
@@ -121,17 +121,20 @@ func (p *AbstractPlugin) GetName() string {
 }
 
 // Init initializes the plugin by calling InitAbc
-func (me *AbstractPlugin) Init() error {
-	return me.InitAbc()
+func (p *AbstractPlugin) Init() error {
+	return p.InitAbc()
 }
 
 // InitAbc initializes the plugin
-func (me *AbstractPlugin) InitAbc() error {
+func (p *AbstractPlugin) InitAbc() error {
 
-	if me.Name = me.Params.GetNameS(); me.Name == "" {
+	if p.ParentParams != nil {
+		p.ParentParams.NewChildS("poller_name", p.Options.Poller)
```

Comment:
where is this poller_name used?

Reply:
https://github.com/NetApp/harvest/blob/030cf8e79f600787dec7e45e2cfb598cd21aaa8b/pkg/conf/conf.go#L382